### DISCUSSION TITLE
RAK4631 is not possible to pair via Bluetooth

### DISCUSSION THREAD

**Discussion by brekovcan** [meshtastic/firmware#4550] [Troubleshooting]
Hi, my setup is based on  RAK4631: RAK19007 Base Board - RAK13800 Ethernet Module - RAK19018 PoE module

After mounting and connecting to MACBook via USBC I did factory reset and upgrade with latest Meshtastic beta firmware. I can setup RAK4631 via Web interface and meshtastic CLI. I set also static PIN for bluetooth. Problem is, that I cannot pair RAK4631 via Bluetooth with iPhone App or MacBook. I can see RAK4631 in device list in App, by clicking on it I can see just Connection Attempts with no success. After first attempt I can see that RAK4631 is immediately rebooted. I think it must be hardware problem with RAK4631. I have also tried to use different firmwares with same result.

**Comment by jp-bennett** [comment]
You probably need to unpair the device in your phone's OS, then try to reconnect

**Reply by brekovcan** [reply]
But this is new device and i have never successfully BT pair with this device. I can see reboot of device immediately if i try to pair from iphone (Meshtastic app) or macbook (client.meshtastic.org).

**Reply by jp-bennett** [reply]
OK, my mistake. I read your description quickly, and assumed it was the more common problem.

**Reply by brekovcan** [reply]
Do you have any idea how to troubleshoot it? I have tried several firmwares, also lates preview, but same behaviour.

**Comment by fifieldt** [comment]
Did you ever track this down, @brekovcan ?

**Reply by brekovcan** [reply]
In my case it was really HW issue with wisblock RAK4631 module, so after RMA replacement, problem was fixed.

**Reply by fifieldt** [reply]
Thanks for letting us know. Glad you got it fixed!

**Comment by tmarkson** [comment]
I experiencing the symptoms in this issue, cannot detect nodes via Bluetooth.  I have 3 different RAK4631 with the same behavior and tried 2.6.11 and 2.7.11. 
Have tried removing from phone. The phone bluetooth does find other devices.  
Any suggestions?

**Reply by tmarkson** [reply]
this is resolved in 2.7.15, not sure of the cause but I can connect to the 4 nodes that were not connecting to phone in 2.7.11
